jobs/log_parser/spec

```diff
@@ -52,6 +52,9 @@ properties:
   logstash_parser.use_local_elasticsearch:
     description: "Run a local elasticsearch client node"
     default: true
+  logstash_parser.plugins:
+    description: "Plugins to run logstash with (array[] = { plugin-name: install-source }"
```

Comment:
@dpb587 - this description `(array[] = { plugin-name: install-source }` confuses me.  Is it possible to give a more verbose description?, eg:
```
  # values are arrays, first being the installed plugin name, second being install source
  logstash_parser.plugins:
    # one of the built-in, official plugins
    - [ "logstash-filter-punct", "logstash-filter-punct" ]
    # an internal plugin
    - [ "cityindex-logstash-filter-acctlookup", "https://s3.amazonaws.com/.../cityindex-logstash-filter-acctlookup-1.2.9.gem" ]
```